### DISCUSSION TITLE
Can mxf files and their xml, indexes be preserved?

### DISCUSSION THREAD

**Discussion by voolstp** [mifi/lossless-cut#1707] [Q&A]
I have a folder of mxf files from Canon R5C camera, that Canon's XF Utility makes when transferring from camera card, and it can see everything in there after it's on disc.
When I export a Segment from LosslessCut, no accompanying .xml file is made, and the Index.mif in the folder is unaffected. So XF Utility does not see the new file.
Is there a configuration that updates these?
I am generally on to Premiere Pro anyway, after using XF Utility to initially to transfer files, review them and delete bad shots. But just wondering.

**Comment by mifi** [comment]
TBH I don't know what is an "accompanying .xml file". From what I understand, [MXF](https://en.wikipedia.org/wiki/Material_Exchange_Format) is a container format and you have files in this format. Not sure where xml files come in, and what they are used for. But I can say that LosslessCut does not support any XML files other than importing segments from certain XML file formats. LosslessCut does not write any XML files
